src/BizHawk.Emulation.Cores/Consoles/Sega/gpgx64/GPGX.ISettable.cs

```diff
@@ -263,7 +263,17 @@ public class GPGXSyncSettings
 			[DefaultValue(LibGPGX.Region.Autodetect)]
 			public LibGPGX.Region Region { get; set; }
 
-			[DisplayName("FM Sound Chip Type")]
+			[DisplayName("[SMS/GG] Load BIOS")]
+			[Description("Indicates whether to load the system BIOS rom.")]
+			[DefaultValue(false)]
+			public bool loadBIOS { get; set; }
+
+			[DisplayName("[SMS] FM Sound Chip Type")]
+			[Description("Sets the method used to emulate the FM synthesizer (main sound generator) of the SMS. 'MAME' is fast and runs full speed on most systems.'Nuked' is cycle accurate, very high quality, and have substantial CPU requirements.")]
+			[DefaultValue(LibGPGX.InitSettings.SMSFMSoundChipType.YM2413_MAME)]

```

Comment:
FM synth is not the main sound generator on SMS. The correct first sentence would be `Sets the method used to emulate the FM Sound Unit of the Sega Mark III/Master System.`